src/CMakeLists.txt

```diff
@@ -135,17 +135,17 @@ set(Mapper_Common_SRCS
  
  compass.cpp
 
- file_format.cpp
- file_format_registry.cpp
- file_import_export.cpp
- file_format_native.cpp
- file_format_ocad8.cpp
- file_format_xml.cpp
- 
- fileformats/ocd_file_export.cpp
- fileformats/ocd_file_format.cpp
- fileformats/ocd_file_import.cpp
- fileformats/ocd_types.cpp
+ file_formats/file_format.cpp
+ file_formats/file_format_registry.cpp
+ file_formats/file_import_export.cpp
+ file_formats/file_format_native.cpp
+ file_formats/file_format_ocad8.cpp
+ file_formats/file_format_xml.cpp
+
+ file_formats/ocad/ocd_file_export.cpp
+ file_formats/ocad/ocd_file_format.cpp
+ file_formats/ocad/ocd_file_import.cpp
+ file_formats/ocad/ocd_types.cpp
```

Comment:
Moving out the ocd_types* is okay. I would prefer the ocd_file_* files not to be moved. 

Apart from this, spelling for new files, directories and classes must be "ocd".